server/server_test.go

```diff
@@ -648,6 +648,13 @@ func TestOAuth2CodeFlow(t *testing.T) {
 			})
 			defer httpServer.Close()
 
+			policy, err := NewRefreshTokenPolicyFromConfig(s.logger, false, "", "", "")
```

Comment:
how about making this a part of newTestServer not to duplicate code?

Reply:
Fixed.